### PR TITLE
Fixed bug that can occur when user has multiple gallery apps

### PR DESCRIPTION
**Pull request by johanforslund**
- [x] Explain the **motivation** for making this change.
- [x] Provide a **test plan** demonstrating that the code is solid.
- [x] Match the **code formatting** of the rest of the codebase.
- [x] Target the `master` branch, NOT a "stable" branch.

## Motivation (required)

Targeting #1017. This bug occurs on Android when user has multiple gallery apps installed on the phone and tries to choose from library. It seems to only occur when launchMode is set to 'singleInstance' in Android Manifest.

This fix starts an Intent Chooser which solves this problem when using 'singleInstance'.

## Test Plan (required)

Tried running it on Android Pie physical phone and on emulator.

### CODE REVIEW

**Comment by mariot** [comment]
Hi!

Any news on this?

Thanks

File: android/src/main/java/com/imagepicker/ImagePickerModule.java [hunk 1]
```diff
@@ -347,7 +347,7 @@ public void launchImageLibrary(final ReadableMap options, final Callback callbac
 
     try
     {
-      currentActivity.startActivityForResult(libraryIntent, requestCode);
+      currentActivity.startActivityForResult(Intent.createChooser(libraryIntent, "Choose gallery app"), requestCode);
```
Comment (Johan-dutoit) [inline android/src/main/java/com/imagepicker/ImagePickerModule.java]: Change "Choose gallery app" to come from the options instead and then I can merge this.

**Comment by johanforslund** [comment]
Options now have a new option called 'chooseWhichLibraryTitle'. If not set, it uses the default title from OS.

**Comment by Johan-dutoit** [comment]
Thanks!  Now just needs to be documented (https://github.com/react-native-community/react-native-image-picker/blob/master/docs/Reference.md#options) and added to the types (https://github.com/react-native-community/react-native-image-picker/blob/master/src/internal/types.ts#L34).

**Comment by johanforslund** [comment]
Fixed!

**Comment by mariot** [comment]
Great news! Thanks a lot

**Pull request merged**